src/shared/services/commandInterpreterHelper.js

```diff
@@ -722,41 +725,66 @@ const availableCommands = [
             result: grassData
           })
         )
-      } else if (param === 'reset') {
-        put(updateGraphStyleData(null))
-      } else if (isValidURL(param)) {
-        if (!param.startsWith('http')) {
-          param = `http://${param}`
-        }
+      }
+
+      function updateAndShowGrass(newGrass) {
+        put(updateGraphStyleData(newGrass))
+        showGrass()
+      }
+
+      function putGrassErrorFrame(message) {
+        put(
+          frames.add({
+            useDb: getUseDb(store.getState()),
+            ...action,
+            error: createErrorObject(InvalidGrassError, message),
+            type: 'error'
+          })
+        )
+      }
+
+      if (param === '') {
+        showGrass()
+        return
+      }
 
+      if (param === 'reset') {
+        updateAndShowGrass(null)
+        return
+      }
+
+      if (
+        isValidURL(param) &&
+        param.includes('.') /* isValid url considers words like rest an url*/
```

Comment:
is this comment suggesting `isValid url considers words like "rest" a url`?

Reply:
yes, It would be clearer if i added quotes perhaps? Even better would be to update the isValidURL to use a better regex though. It was used in quite a lot of places so I wasn't comfortable doing the change when I wrote this. I could look into it now though